src/components/auth/SettingsModal.tsx

```diff
@@ -113,9 +114,28 @@ export const SettingsMenuItem = React.memo(function SettingsMenuItem({ finalRef
         setShowMoxfieldLinker(!showMoxfieldLinker);
     }, [showMoxfieldLinker]);
 
+    const onMoxfieldInputBlur = useCallback(async () => {
+        const moxfieldProfileObj: MoxfieldProfile | undefined = await getMoxfieldProfile(moxfieldIdInputValue);
+        console.log(moxfieldProfileObj); // TODO: Remove before merge
+
+        if (moxfieldProfileObj === undefined) {
+            setMoxfieldImageUrl(errorMoxfieldLogo);
+            setMoxfieldImageValidated(false);
+            return;
+        }
+
+        if (moxfieldProfileObj.imageUrl) {
+            setMoxfieldImageUrl(moxfieldProfileObj.imageUrl);
+            setMoxfieldImageValidated(true);
+        } else {
+            setMoxfieldImageUrl(missingMoxfieldProfileImage);
+            setMoxfieldImageValidated(false);
```

Comment:
does this mean that if a user is missing a moxfield imageUrl, then you cannot save? what about a user who did not set their image in moxfield?